### PR TITLE
Add delete option to word forms

### PR DESCRIPTION
**Pull request by djrenren**

### CODE REVIEW

**Review by 7imbrook — DISMISSED** [review]

File: client/src/reducers/lessonReducer.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@ export interface WordInfo {
     word: string;
     part_of_speech?: string;
     forms?: WordForm[];
+    synonyms?: string[];
+    antonyms?: string[];
+    definition?: string;
```
Comment (7imbrook) [inline client/src/reducers/lessonReducer.ts:12]: Can you fully type this out?

**Review by 7imbrook — CHANGES_REQUESTED** [review]

File: client/src/reducers/lessonReducer.ts (hunk 1 above)
Comment (djrenren) [reply client/src/reducers/lessonReducer.ts:12]: It's fully typed out for the set of implemented features... figured I'd expand it when we get context sentences. Is that what you meant?

Comment (7imbrook) [reply client/src/reducers/lessonReducer.ts:12]: No I mean typed out for the server response as is.

Comment (djrenren) [reply client/src/reducers/lessonReducer.ts:12]: I'd normally do that in another commit but yeah I can just add that here.

**Review by 7imbrook — APPROVED** [review]

**Pull request merged**